util/ci_utils.sh

```diff
@@ -249,8 +249,8 @@ test_wheel() {
     python -m pip --version
     echo "Installing Open3D wheel $wheel_path in virtual environment..."
     python -m pip install "$wheel_path"
-    python -c "import open3d; print('Installed:', open3d); print('BUILD_CUDA_MODULE: ', open3d._build_config['BUILD_CUDA_MODULE'])"
-    python -c "import open3d; print('CUDA available: ', open3d.core.cuda.is_available())"
+    python -W default -c "import open3d; print('Installed:', open3d); print('BUILD_CUDA_MODULE: ', open3d._build_config['BUILD_CUDA_MODULE'])"
```

Comment:
Why do we need -W default now?

Reply:
Extra debug warnings, if importing open3d fails.